recipes/cm-unicode/bld.bat

```diff
@@ -0,0 +1,4 @@
+mkdir %PREFIX%/fonts
```

Comment:
You don't need this file.  `noarch: generic` means the package will only be built on Linux, but installable everywhere.

Reply:
But it's good to have one here for anybody who wants to build from source on windows.

Reply:
fair